collector/mssql.go

```diff
@@ -19,18 +19,53 @@
 package collector
 
 import (
+	"errors"
 	"fmt"
+	"regexp"
+	"sync"
+	"time"
 
 	"github.com/StackExchange/wmi"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"golang.org/x/sys/windows/registry"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
```

Comment:
The package name is `kingpin` anyway, no need to alias.

Reply:
That was VSCode trying to be helpful.  I'll remove that.